official/mnist/mnist_eager_test.py

```diff
@@ -76,5 +82,4 @@ def test_evaluate_with_defun(self):
 
 
 if __name__ == "__main__":
-  tfe.enable_eager_execution()
```

Comment:
should we also remove the tfe import? then it will work in 2.0 as well (maybe?) or are you mainly trying to get it working with 1.x?